src/test/java/org/minions/devfund/yuri/WackAMole/WackAMoleTest.java

```diff
@@ -1,7 +1,8 @@
-package org.minions.devfund.yuri;
+package org.minions.devfund.yuri.WackAMole;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)